enterprise_subsidy/apps/transaction/management/commands/backpopulate_transaction_course_run_start_date.py

```diff
@@ -0,0 +1,55 @@
+import logging
+from django.core.management.base import BaseCommand
+from openedx_ledger.models import Transaction
+from enterprise_subsidy.apps.content_metadata.api import ContentMetadataApi
+from enterprise_subsidy.apps.subsidy.models import Subsidy
+from dateutil import parser
+from enterprise_subsidy.apps.transaction.utils import normalize_to_datetime
+
+logger = logging.getLogger(__name__)
+
+class Command(BaseCommand):
+    help = "Backpopulating course_run_start_date for Transactions by fetching from content metadata."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--since',
+            type=str,
+                help='Only update transactions modified on or after this date (ISO format, e.g. 2025-09-01)',
+            required=False,
+        )
+
+    def handle(self, *args, **options):
+        content_api = ContentMetadataApi()
+        transactions = Transaction.objects.filter(course_run_start_date__isnull=True, content_key__isnull=False).exclude(content_key='')
```

Comment:
[question] Were you able to determine if it’s feasible to run this job in batches, processing only a limited number of records per iteration to help avoid failures?

Reply:
@jono-booth We have some Enterprise cron jobs set up in Jenkins and a few in ArgoCD but I checked the for this repo you'll need to set it up in [ArgoCD](https://argocd.tools.edx.org/applications/prod-enterprise-subsidy). I haven't really configured a job here before, it might be worth checking with Titans or probably someone from SRE before doing the setup.